week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -14,9 +14,36 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 "think" like a computer and practice your knowledge of basic JavaScript.
 */
 
+
 function sortAges(arr) {
+  
+  let arr2 = []; //blank array
+  for (i = 0; i < arr.length; i++) {
+
+    if (typeof arr[i] === "number") {
+      arr2.push(arr[i]) //only numbers are pushed into arr2
+      }
+  }

```

Comment:
Nice! Very good!

2 small things to consider next time to push yourself a bit:
- Could we extract a little function out of this? Taking parts of code and extracting them into named functions can improve how easy it is to read the code and you could perhaps re-use it later. A place like this could be a good example where you can extract a nice self-contained part of the code. 
- Be careful about naming variables as `arr2`. In short functions like this it might still be ok, but try to think about a more descriptive name. This will be very important later when you are working on longer pieces of code!